tests/drink/src/psp22_tests.rs

```diff
@@ -0,0 +1,77 @@
+use anyhow::Result;
+use assert2::assert;
+use drink::{runtime::MinimalRuntime, session::Session};
+use ink_primitives::AccountId;
+use ink_wrapper_types::{util::ToAccountId, Connection};
+use psp22_contract::{Instance, PSP22 as _};
+
+use crate::*;
+
+pub fn balance_of(
+    session: &mut Session<MinimalRuntime>,
+    psp22: Instance,
+    account: AccountId,
+) -> u128 {
+    session
+        .query(psp22.balance_of(account))
+        .unwrap()
+        .result
+        .unwrap()
+}
+
+pub fn setup(session: &mut Session<MinimalRuntime>, caller: drink::AccountId32) -> Instance {
+    let _code_hash = session.upload_code(psp22_contract::upload()).unwrap();
+
+    let _ = session.set_actor(caller);
+
+    session
+        .instantiate(Instance::new(1000))
+        .unwrap()
+        .result
+        .to_account_id()
+        .into()
+}
+
+#[test]
+fn test_transfers() -> Result<()> {
+    let mut session: Session<MinimalRuntime> = Session::new().expect("Init new Session");
```

Comment:
just a low nit: sometimes, it will spare you one line:
`let mut session = Session::<MinimalRuntime>::new().expect("Init new Session");`

Reply:
Might be that my IDE added it automatically.

Reply:
Removed.